app/models/reports/region_summary_schema.rb

```diff
@@ -325,45 +325,57 @@ def initialize(regions, periods:)
       end
     end
 
+    memoize def patients_called_breakdown_rates(entry)
```

Comment:
```suggestion
    memoize def patients_called_breakdown_rates(slug, period)
```

The `entry` var's meaning is lost once it's outside the loop it's called in.

Reply:
Right, will make the change

Reply:
it's done ✅